CHANGELOG.md

```
@@ -1,5 +1,9 @@
 # Changelog
 
+## Ongoing
+
+PR []()
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)